### PR TITLE
Updating change_scene and change_scene_to

### LINKED ISSUE #6156: `change_scene` and `change_scene_to` have been renamed
**Your Godot version:**
v4.0.alpha.custom_build [d2a6a187f]

**Issue description:**
Now that godotengine/godot#63596 is merged, `change_scene` needs to be replaced with `change_scene_to_file` and `change_scene_to` needs to be replaced with `change_scene_to_packed`.

**URL to the documentation page:**
<https://docs.godotengine.org/en/latest/tutorials/scripting/scene_tree.html?highlight=change_scene#changing-current-scene>

### PR DESCRIPTION
**Pull request by Kirrby**
Closes #6156

### CODE REVIEW

File: tutorials/scripting/scene_tree.rst [hunk 1]
```diff
@@ -141,14 +141,14 @@ Changing current scene
 
 After a scene is loaded, you may want to change this scene for
 another one. One way to do this is to use the
-:ref:`SceneTree.change_scene() <class_SceneTree_method_change_scene>`
+:ref:`SceneTree.change_scene_to_file() <class_SceneTree_method_change_scene>`
```
Comment (Jayman2000) [inline tutorials/scripting/scene_tree.rst:144]: You also need to update the `<class_SceneTree_method_…>` part or else the link won’t work.

File: tutorials/scripting/scene_tree.rst [hunk 2]
```diff
@@ -160,15 +160,15 @@ function:
 Rather than using file paths, one can also use ready-made
 :ref:`PackedScene <class_PackedScene>` resources using the equivalent
 function
-:ref:`SceneTree.change_scene_to(PackedScene scene) <class_SceneTree_method_change_scene_to>`:
+:ref:`SceneTree.change_scene_to_packed(PackedScene scene) <class_SceneTree_method_change_scene_to>`:
```
Comment (Jayman2000) [inline tutorials/scripting/scene_tree.rst:163]: Ditto.

**Review by Jayman2000 — CHANGES_REQUESTED** [review]
This PR introduces two warnings:

<pre><samp>/home/jayman/Documents/Home/VC/Partially mine/godot-docs/repo/tutorials/scripting/scene_tree.rst:142: WARNING: undefined label: class_scenetree_method_change_scene
/home/jayman/Documents/Home/VC/Partially mine/godot-docs/repo/tutorials/scripting/scene_tree.rst:160: WARNING: undefined label: class_scenetree_method_change_scene_to
</samp></pre>

If you fix those links, the warnings should go away.

**Comment by YuriSizov** [comment]
You also have a typo in your commit message, which is worth fixing when you'll be rebasing this PR after applying the suggested changes and fixes.

**Comment by Kirrby** [comment]
ah damn didnt see that, will fix

**Comment by Kirrby** [comment]
ok sorry for not being the smartest (im very new to github and all that) i made the changes on my repo, how do i add them to this pr? i also made the changes directly on the browser editor so i think that doesnt help at all.

**Comment by YuriSizov** [comment]
> ok sorry for not being the smartest (im very new to github and all that) i made the changes on my repo, how do i add them to this pr? i also made the changes directly on the browser editor so i think that doesnt help at all.

If you make changes to the same branch, they will just appear here. If you've made a new branch, then this won't work. You also can't rebase or squash using the GitHub's web UI and need to use a desktop client for Git to do that.

**Comment by Kirrby** [comment]
ah, okay, imma make the changes on the right branch

**Comment by Kirrby** [comment]
ok changes made

**Comment by Kirrby** [comment]
i think that should be all but i'll check

**Comment by Jayman2000** [comment]
I found two more places that need to be updated:

```console
$ git grep -P 'change_scene(?!_to_(file|packed))'
[…]
tutorials/scripting/singletons_autoload.rst:        # Optionally, to make it compatible with the SceneTree.change_scene() API.
tutorials/scripting/singletons_autoload.rst:        // Optionally, to make it compatible with the SceneTree.change_scene() API.
```

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thanks and congrats on your first merged PR to Godot Engine's documentation! And thanks for everyone helping to point out things :)

**Comment by Kirrby** [comment]
> Thanks and congrats on your first merged PR to Godot Engine's documentation! And thanks for everyone helping to point out things :)

Thanks!

**Pull request merged**